src/mono/wasm/build/WasmApp.Common.targets

```diff
@@ -804,7 +804,7 @@
     Outputs: file @(_WasmCompileBitcodeRsp
   -->
   <Target Name="_WasmWriteRspForCompilingBitcode">
-    <WriteLinesToFile Lines="@(_BitcodeLDFlags)" File="$(_WasmCompileBitcodeRsp)" Overwrite="true" WriteOnlyWhenDifferent="true" />
+    <WriteLinesToFile Lines="@(_EmccCommonFlags)" File="$(_WasmCompileBitcodeRsp)" Overwrite="true" WriteOnlyWhenDifferent="true" />
```

Comment:
the comment above still references `_BitcodeLDFlags`.

I'm also lacking enough context to check why `_EmccCommonFlags` is enough 😄

Reply:
updated the comment. also thinking more about it, I replaced the original _BitcodeLDFlags with _BitcodeCompileFlags. that way it is more clear and we still provide way to inject extra flags for BC compilation

Reply:
the previously used `_EmccLDFlags` are based on `_EmccCommonFlags` plus linking specific flags. similar for the `_WasiClangLDFlags`